app/models/manageiq/providers/nutanix/inventory/collector/infra_manager.rb

```diff
@@ -1,90 +1,38 @@
 class ManageIQ::Providers::Nutanix::Inventory::Collector::InfraManager < ManageIQ::Providers::Nutanix::Inventory::Collector
-  def connection
-    port = manager.port || 9440
-    @connection ||= ManageIQ::Providers::Nutanix::InfraManager.raw_connect(
-      manager.hostname,
-      port,
-      manager.authentication_userid,
-      manager.authentication_password,
-      manager.verify_ssl || OpenSSL::SSL::VERIFY_NONE
-    )
-  end
+  require "nutanix_vmm"
+  require "nutanix_clustermgmt"
```

Comment:
Gah - I just noticed, but these cannot be at parse time or they end up loading early in all processes.

Reply:
https://github.com/ManageIQ/manageiq-providers-nutanix/pull/15